bookshelf/app.yaml

```diff
@@ -0,0 +1,12 @@
+runtime: custom
+vm: true
+
+manual_scaling:
```

Comment:
`manual_scaling`? I can remove this. Are we OK for getting-started users to have 2 instances from the start and potentially pay some unnecessary money? I prefer having a `scaling` section in the guide to show how to remove this setting if necessary?